core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -114,6 +114,7 @@
     public static final String VERTX_WEB = "vertx-web";
     public static final String VERTX_GRAPHQL = "vertx-graphql";
     public static final String WEBJARS_LOCATOR = "webjars-locator";
+    public static final String VERTX_REDIS = "vertx-redis";
```

Comment:
I don't know about the name. The SQL clients went for `Reactive XXX client`. MongoDB is simply `mongodb-client`.

@cescoffier WDYT?

Reply:
After discussion with @cescoffier , let's be consistent with the MongoDB one and name it `redis-client`.